js/blocks/loader/advanced.js

```diff
@@ -0,0 +1,43 @@
+const { Notice } = wp.components;
+const { InspectorAdvancedControls } = wp.editor;
+const { select, dispatch } = wp.data;
+const { __ } = wp.i18n;
+
+const ADDITIONAL_CSS_NOTICE_ID = 'additional_css_class';
+
+const removeNotice = ( event ) => {
+	event.currentTarget.parentNode.remove()
+	dispatch( 'block-lab' ).setStatus( ADDITIONAL_CSS_NOTICE_ID, 'dismissed' );
+}
+
+const advancedControls = () => {
+
+	const notice = () => {
+		let status = select( 'block-lab' ).getNotice( ADDITIONAL_CSS_NOTICE_ID );
+
+		if ( 'dismissed' === status ) {
+			return
+		}
+
+		return (
+			<Notice
+				status="info"
+				isDismissible={true}
+				onRemove={ removeNotice }
+				id="bl-inspector-notice"
+				className="bl-inspector-notice"
+			>
+				<p>{sprintf( __( 'Include the Additional CSS Class in a block template by using this field:', 'block-lab' ), '' )}<br /><code>className</code></p>
```

Comment:
This text could be better, but there's not really a way of including the `<code>` tags in the `sprintf` (since it returns a string, and we need a JSX element). So, unfortunately, the `<code>className</code>` but _needs_ to be right at the end of the sentence.

Reply:
Sure, that's fine. Is `sprintf()` needed here, though? 

It looks like its second argument is `''`, so it doesn't do any interpolation.

Reply:
Ha! Whoops. That was leftover from my trying to get the <code> part to work. Removed in b3d30f4.

Reply:
All good 😄